### PR TITLE
{tools}[GCC/4.9.2] Emacs and Doxygen (REVIEW)

### PR DESCRIPTION
**Pull request by gjbex**
* Emacs
* Doxygen

### CODE REVIEW

**Comment by gjbex** [comment]
Test report by @gjbex
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux red hat enterprise linux computenode 6.4, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.6
See https://gist.github.com/0062b306fe2da263b621 for a full test report.

**Comment by boegel** [comment]
@wpoely86: please review/test?

File: easybuild/easyconfigs/e/Emacs/Emacs-24.4-GCC-4.9.2.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+name = 'Emacs'
+version = '24.4'
+
+homepage = 'http://www.gnu.org/software/emacs/'
+description = """GNU Emacs is an extensible, customizable text editor—and more.
+ At its core is an interpreter for Emacs Lisp, a dialect of the Lisp programming language
+ with extensions to support text editing."""
+
+toolchain = {'name': 'GCC', 'version': '4.9.2'}
+toolchainopts = {}
```
Comment (wpoely86) [inline easybuild/easyconfigs/e/Emacs/Emacs-24.4-GCC-4.9.2.eb]: drop this if it's not needed

**Comment by wpoely86** [comment]
ok to test

**Comment by wpoely86** [comment]
@boegel Tiny remark, else good to go if Jenkins agrees

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/462cbbd7c8d7fe0329d7 for a full test report.

**Comment by boegel** [comment]
Looks great, going in, thanks @gjbex!

**Pull request merged**